svy/src/helpers/account.ts

```diff
@@ -46,6 +50,9 @@ export function receiveSVY(accountAddress: Address, svyReceived: BigInt, block:
   const oldSVYBalance = account.svyBalance;
   account.svyBalance = account.svyBalance.plus(svyReceived);
   account.svyBalanceUSD = getSVYBalanceInUSD(account.svyBalance);
+  account.svyEarnRatePerSec = SFIAContract.getMySVYPageInfo(accountAddress).svyEarnRatePerSec;
```

Comment:
Cache this value in a variable. Calling functions of a binded contract result in an eth_call. Eth calls are computationally expensive. Call it once then refer to the result as needed.